quickwit-cli/Cargo.toml

```diff
@@ -72,5 +72,6 @@ serial_test = "0.7.0"
 ci-test = []
 tokio-console = ["console-subscriber"]
 openssl-support = ["openssl-probe"]
+simulates3 = ["quickwit-metastore/simulates3"]
```

Comment:
`simulate-s3` seems more readable

Reply:
that will show up with `--all-features`.